include/odp/api/spec/cpu.h

```diff
@@ -115,14 +119,15 @@ uint64_t odp_cpu_hz_max_id(int id);
  * Return current CPU cycle count. Cycle count may not be reset at ODP init
  * and thus may wrap back to zero between two calls. Use odp_cpu_cycles_max()
  * to read the maximum count value after which it wraps. Cycle count frequency
- * follows the CPU frequency and thus may change at any time. The count may
- * advance in steps larger than one. Use odp_cpu_cycles_resolution() to read
- * the step size.
+ * follows the CPU frequency and thus may change at any time. Cycle count should
+ * not be used for time measurements due to the possibility of frequency
+ * variation. The count may advance in steps larger than one. Use
+ * odp_cpu_cycles_resolution() to read the step size.
  *
- * @note Do not use CPU count for time measurements since the frequency may
- * vary.
+ * Returns zero if CPU cycle counter is not supported.
  *
  * @return Current CPU cycle count
+ * @retval 0 Not supported
  */
 uint64_t odp_cpu_cycles(void);
```

Comment:
Matching validation test change is missing.

Reply:
Fixed in v3